libs/framework/src/bundle_context.c

```diff
@@ -559,28 +560,17 @@ long celix_bundleContext_trackBundlesWithOptions(
         const celix_bundle_tracking_options_t *opts) {
     long trackerId = -1;
     celix_bundle_context_bundle_tracker_entry_t *entry = calloc(1, sizeof(*entry));
-    if (entry != NULL) {
```

Comment:
IMO we should not check NULL. Because this is untestable and results in bad coverage / untested exceptional flows. 
Modern OS's always return a value and work with a oom killer.